src/content/configuration/dev-server.md

```diff
@@ -929,7 +932,7 @@ webpack-dev-server --pfx /path/to/file.pfx
 
 `string`
 
-The passphrase to a SSL PFX file.
+The passphrase to an SSL PFX file.
```

Comment:
No sure about this one https://english.stackexchange.com/questions/1016/do-you-use-a-or-an-before-acronyms-initialisms

Reply:
It is subjective. I would go for the `an` version.

Reply:
lets revert as its debatable please